client/knowledge_base_search/src/containers/SearchContainer/styles.css

```diff
@@ -1,11 +1,88 @@
 .header--input {
   color: #ffffff;
   text-align: center;
+  margin-bottom: 2rem;
+}
+
+.clickable-tabs {
+  display: table;
+  border-spacing: 1rem;
+  width: 70rem;
+  position: relative;
+  bottom: 3rem;
+}
+
+.clickable-tab {
```

Comment:
we might want to wait on the visual design specifics before going crazy on the tweaks here